docs/recipes/ember.md

```diff
@@ -0,0 +1,147 @@
+# Usage with Ember
+
+Ember follows a similar pattern to [Vue](./vue.md):
+
+- The machine can be defined externally;
+- The service is placed as a property of the component;
+- State changes are observed via `service.onTransition(state => ...)`, where you set some data property to the next `state`;
+- The machine's context can be referenced as an external data store by the app. Context changes are also observed via `service.onTransition(state => ...)`, where you set another data property to the updated context;
+- The service is started (`service.start()`) when the component is created `constructor()`;
+- Events are sent to the service via `service.send(event)`.
+
+## Modern Ember (Octane)
```

Comment:
Just ember > version 3.13


Though, if getting rid of the other section, than the title isn't needed, I think.

Reply:
Unfortunately there was still a bug in 3.13 that caused issues with `@tracked` (but not in this example) which was fixed in 3.14 and why I focus on that number. Will put 3.13.